desktop/ui/src/main/java/org/datacleaner/widgets/tree/SchemaTree.java

```diff
@@ -670,4 +674,9 @@ public void filter(String searchTerm) {
             }
         }
     }
+
+    public void setIncludeLibraryNode(boolean includeLibraryNode) {
+        _includeLibraryNode = includeLibraryNode;
+    }
```

Comment:
The way we _could_ do it in Guice was to inject a SchemaTreeConfiguration object or so. But I think that's more work and doesn't make anything much clearer.

Another approach would be to break SchemaTree into two classes: A basic one, and one with the Library in it. And then only inject the basic one when needed.

But all in all, I am fine with this "keep it simple, stupid" solution. It's not particularly pretty, but it's also not unlike many other swing components which are generally mutable.